src/emulator/gxm/src/gxm.cpp

```diff
@@ -296,10 +297,14 @@ static std::string generate_vertex_glsl(const SceGxmProgram &program) {
     return glsl.str();
 }
 
-static void dump_missing_shader(const char *hash, const char *extension, const SceGxmProgram &program, const char *source) {
+static void dump_missing_shader(const char *hash, const char *extension, const SceGxmProgram &program, const char *source, const char *base_path, std::string game_title) {
```

Comment:
This should probably be `pref_path` rather than `base_path`. The Base path is inside the app bundle (shared between users), and the pref path is user-specific, inside their home directory.